framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -2530,4 +2535,33 @@ private void printGitProperties() {
             System.out.println("IOException while reading git.properties: " + e.getMessage());
         }
     }
+
+    /**
+     * Extract the version of the Checker Framework
+     *
+     * @throws IOException if unable to read docs/developer/release/release.properties or
+     *     docs/examples/MavenExample/pom.xml file
+     * @throws NullPointerException if tag present in release.properties for Checker Framework
+     *     version is not found in docs/examples/MavenExample/pom.xml
+     * @return Checker Framework version {@link String}
```

Comment:
It is usually not necessary to state the type, which is already stated nearby in the source code and in the generated API documentation.